text/0000-CNB-user-research-2021.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: CNB User Research 2021
+- Start Date: 2020-12-10
+- Author(s): sampeinado, speinado@vmware.com
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Background
+The purpose of this study is to understand the major pain points buildpack authors and app developers experience in using Cloud Native Buildpacks and how these issues slow the adoption of CNBs as an emerging standard in containerization. VMware has donated funding for the recruitment and compensation of 15-20 interview subjects to help the CNB project grow. 
+ 
+Given our contributors’ strong understanding of buildpacks and their spec, the project is at risk of a blindspot towards novice buildpack authors who want to create value quickly. The CNB spec enables more modularity than the v2 spec, but its API is also significantly more complex. For our ecosystem of buildpacks to grow, and therefore the number of app types and use cases buildpacks support to grow, we need to develop a better understanding of where buildpack authors struggle and how we can support them. 
+ 
+In addition, we want to better understand why app developers switch to CNBs and what challenges they face when doing so. As a project, we have clear assumptions about why CNBs are valuable relative to other containerization tools, but we don’t know which reasons are most important to which users. Without this clarity, our ability to market the project’s strengths and prioritize the most valuable features is limited. 
+
+# Objectives
+<ol>
```

Comment:
nit: can we convert the html here to a markdown ordered list so it is readable in it's plain text format?

Reply:
i didnt do this bc i wasn't sure how to nest the sub items using markdown

Reply:
You create a nested list you can indent the nested items. E.g. 

1. Understand how buildpack authors currently go about writing buildpacks and customizing builds and what goals they are trying to accomplish by doing so
    1. Evaluate pain points buildpack authors are experiencing </li>
    1. Evaluate the relative usability of proposals like [packfile](https://github.com/sclevine/packfile or [inline buildpacks](https://github.com/kr/heroku-buildpack-inline)
1. Define the different segments of app developers using buildpacks, their goals, challenges, and motivations to switch to buildpacks from other tools

is rendered from 
```
1. Understand how buildpack authors currently go about writing buildpacks and customizing builds and what goals they are trying to accomplish by doing so
    1. Evaluate pain points buildpack authors are experiencing </li>
    1. Evaluate the relative usability of proposals like [packfile](https://github.com/sclevine/packfile or [inline buildpacks](https://github.com/kr/heroku-buildpack-inline)
1. Define the different segments of app developers using buildpacks, their goals, challenges, and motivations to switch to buildpacks from other tools
```